Distinguishing entry "ident" from entry "name" in term notations. (#11841)
coq/coq: doc/changelog/03-notations/11841-master+distinguishing-ident-name-in-grammar-entries.rst

- **Changed:**
  In notations (except in custom entries), the misleading :n:`@syntax_modifier`

>   :n:`@ident ident` (which was standing for parsing either an identifier or

Reviewer: ```suggestion
  :n:`@ident ident` (which accepted either an identifier or
```

>   In notations (except in custom entries), the misleading modifier
>   :g:`x ident` (which was standing for parsing either an identifier or
>   a :g:`_`) is deprecated and should be replaced by :g:`x name`. If

Reviewer 2: ```suggestion
  In notations (except in custom entries), the misleading :n:`@syntax_modifier`
  :n:`@ident ident` (which was standing for parsing either an identifier or
  a :g:`_`) is deprecated and should be replaced by :n:`@ident name`. If
```

  the intent was really to only parse identifiers, this will

>   eventually become possible, but only as of Coq 8.14.

Reviewer 2: Here you say that it will become possible as of Coq 8.14 but in your updated plan this was rather Coq 8.15. Personally I'm OK with skipping the intermediate fatal warning step, but to be on the safe side, don't you want to say "in a future version" instead of making predictions?

>   eventually become possible, but only from the next version of Coq.

Reviewer: ```suggestion
  eventually become possible, but only as of Coq 8.14.
```

>   In custom entries, the meaning of :g:`x ident` is silently changed

Reviewer 2: ```suggestion
  In custom entries, the meaning of :n:`@ident ident` is silently changed
```
